### PR TITLE
SDK-290: Should be possible to specify `openmrs` frontend version

### PR DESCRIPTION
**Pull request by brandones**
https://issues.openmrs.org/browse/SDK-290

### CODE REVIEW

**Comment by brandones** [comment]
I don't understand that build failure. I'll look more on Monday.

File: integration-tests/src/test/java/org/openmrs/maven/plugins/AbstractSdkIntegrationTest.java [hunk 1]
```diff
@@ -23,9 +24,11 @@
 
 import java.io.File;
 import java.io.FileInputStream;
+import java.io.FileNotFoundException;
```
Comment (ibacher) [inline integration-tests/src/test/java/org/openmrs/maven/plugins/AbstractSdkIntegrationTest.java:27]: Seems to be an unused import?

File: integration-tests/src/test/java/org/openmrs/maven/plugins/AbstractSdkIntegrationTest.java [hunk 2]
```diff
@@ -229,6 +234,15 @@ public void assertFilePresent(String... paths) {
         assertPathPresent(resolvedPath);
     }
 
+    public void assertFileContains(String regex, String... paths) throws IOException {
+        Path resolvedPath = testDirectoryPath.toAbsolutePath();
+        for (String path : paths) {
+            resolvedPath = resolvedPath.resolve(path);
+        }
+        String jsContents = new String(Files.readAllBytes(resolvedPath));
```
Comment (ibacher) [inline integration-tests/src/test/java/org/openmrs/maven/plugins/AbstractSdkIntegrationTest.java:242]:
```suggestion
        String jsContents = new String(Files.readAllBytes(resolvedPath, StandardCharsets.UTF_8));
```

File: integration-tests/src/test/java/org/openmrs/maven/plugins/DeployIntegrationTest.java [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package org.openmrs.maven.plugins;
 
+import org.hamcrest.Matchers;
```
Comment (ibacher) [inline integration-tests/src/test/java/org/openmrs/maven/plugins/DeployIntegrationTest.java:3]: I think all the added imports in this file can be removed, right?

**Review by ibacher — COMMENTED** [review]
I think there are a couple of very small things to adjust, but otherwise, this LGTM. I wouldn't worry too much about the build error... it's something to do with the overhaul of the `createProject()` implementation which is unrelated to this PR.

**Comment by brandones** [comment]
Thanks @ibacher ! Fixed as requested.

**Review by ibacher — APPROVED** [review]

**Pull request merged**